draft-ietf-intarea-provisioning-domains.md

```diff
@@ -622,16 +635,26 @@ fetching the updated object is calculated as a uniformly random time
 in the interval \[(B-A)/2,B\].
 
 In the example {{pvd_example}}, the delay field value
-is 5, this means that the host calculates its delay by choosing a random
-number between 0 and 2**(5 * 2) milliseconds, i.e., between 0 and 1024
+is 1, this means that the host calculates its delay by choosing a uniformly
+random time between 0 and 2**(10 + 1) milliseconds, i.e., between 0 and 2048
 milliseconds.
 
 Since the 'Delay' value is directly within the PvD Option rather
 than the object itself, an operator may perform a push-based update by
-incrementing the Sequence value while changing the Delay value
+incrementing the Sequence Number value while changing the Delay value
 depending on the criticality of the update and its PvD Additional
 Information servers capacity.
 
+In addition to adding a random delay when fetching Additional Information, hosts
+MUST enforce a minimum time between requesting Additional Information
+for a given PvD on the same network. This minimum time is RECOMMENDED
+to be 10 seconds, in order to avoid hosts causing a denial-of-service on the
+PvD server. Hosts also MUST limit the number of requests that are made to
+different PvD Additional Information servers on the same network within a short
+period of time. A RECOMMENDED value is to issue no more than five PvD
+Additional Information requests in total on a given network within 10 seconds.
+For more discussion, see {{security}}.
```

Comment:
plural form for "discussion" ?

Reply:
I think singular is fine here, grammatically.

Reply:
You are the native speaker :-) You will need to explain me once why 'more' can be followed by a singular noun.

Reply:
Take a look at definition here: https://dictionary.cambridge.org/us/dictionary/english/discussion

As a noun, "discussion" can be either C (countable) or U (uncountable) ([C/U]). Thus, like how you can have "more water" you can also have "more discussion".